tests/config.c

```diff
@@ -230,6 +234,23 @@ config_parse(master_config *cfg, const uint8 num_config, int argc, char *argv[])
                cfg[cfg_idx].use_log = TRUE;
             }
          }
+         config_has_option("no-log")
+         {
+            for (cfg_idx = 0; cfg_idx < num_config; cfg_idx++) {
+               cfg[cfg_idx].use_log = FALSE;
+            }
+         }
+         config_has_option("verbose-logging")
```

Comment:
This name is fine. Just fyi, that I have issue #353 open for something very similar. For that, then, I will name the option `--verbose-progress` as you have reserved this name.

Reply:
Ack.